wp-content/plugins/core/src/Blocks/Types/Interstitial/Interstitial.php

```diff
@@ -22,6 +22,9 @@ class Interstitial extends Block_Config {
 	public const LAYOUT_LEFT      = 'left';
 	public const LAYOUT_CENTER    = 'center';
 
+	/**
+	 * @return void
+	 */
 	public function add_block() {
```

Comment:
Would be way better to just add `add_block(): void {` to all of these remove the @return docBlock. We don't need return comments if the method specifies what they are unless it's an array and then we should specify what type of array it is.